### PR TITLE
sriov pkg: WithConfigDaemonNodeSelector method added

### PR DESCRIPTION
**Pull request by elenagerman**
WithConfigDaemonNodeSelector method added

### CODE REVIEW

**Review by kononovn — DISMISSED** [review]
/lgtm

File: pkg/sriov/operatorconfig.go [hunk 1]
```diff
@@ -130,13 +130,13 @@ func (builder *OperatorConfigBuilder) Exists() bool {
...
 
-// WithInjector configures enableInjector in the SriovOperatorConfig.
+// WithInjector configures replace in the SriovOperatorConfig.
 func (builder *OperatorConfigBuilder) WithInjector(enable bool) *OperatorConfigBuilder {
 	if valid, _ := builder.validate(); !valid {
 		return builder
 	}
 
-	glog.V(100).Infof("Configuring enableInjector %t to SriovOperatorConfig object %s",
+	glog.V(100).Infof("Configuring replace %t to SriovOperatorConfig object %s",
```
Comment (achuzhoy) [inline pkg/sriov/operatorconfig.go:139]:
```suggestion
	glog.V(100).Infof("Configuring replace %t in SriovOperatorConfig object %s",
```

File: pkg/sriov/operatorconfig.go [hunk 2]
```diff
@@ -160,6 +160,40 @@ func (builder *OperatorConfigBuilder) WithOperatorWebhook(enable bool) *Operator
...
 }
 
+// WithConfigDaemonNodeSelector configures configDaemonNodeSelector in the SriovOperatorConfig.
+func (builder *OperatorConfigBuilder) WithConfigDaemonNodeSelector(
+	configDaemonNodeSelector map[string]string) *OperatorConfigBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Configuring configDaemonNodeSelector %s to SriovOperatorConfig object %s in namespace %s",
```
Comment (achuzhoy) [inline pkg/sriov/operatorconfig.go:170]:
```suggestion
	glog.V(100).Infof("Configuring configDaemonNodeSelector %s in SriovOperatorConfig object %s in namespace %s",
```

File: pkg/sriov/operatorconfig.go (hunk 1 above)
Comment (elenagerman) [reply pkg/sriov/operatorconfig.go:139]: sure, thank you! I have no idea why it was changed at all

**Review by achuzhoy — APPROVED** [review]

**Pull request merged**